### PR TITLE
Add JUnit result summaries to the build summary page

### LINKED ISSUE #172: Gubernator should also show passed and skipped tests
It can be helpful for determining whether a run passed because all tests actually passed or if many tests were skipped.

This can probably just be a smaller list at the bottom underneath any failures.

### PR DESCRIPTION
**Pull request by smarterclayton**
Fixes #172

Built on top of #1981 because signature of parse junit changes

### CODE REVIEW

**Comment by smarterclayton** [comment]
@kubernetes/sig-testing-pr-reviews

**Comment by smarterclayton** [comment]
Note a slight tweak from the screenshot in the issue - failures are first (and red), successes are second (uncolored), then skipped, then total.

File: gubernator/view_build.py [hunk 1]
```diff
@@ -28,8 +28,11 @@
 import view_base
 
 
-def parse_junit(xml, filename):
-    """Generate failed tests as a series of (name, duration, text, filename, output) tuples."""
+def parse_junit(xml, filename, stats):
```
Comment (rmmh) [inline gubernator/view_build.py]: This is a weird function signature, it would probably be best to refactor this into a class later.

**Pull request merged**